docs/_docs/plugins.md

```diff
@@ -58,6 +58,21 @@ You have 3 options for installing plugins:
 
 <div class="note info">
   <h5>
+    The <code>:jekyll_plugins</code> group in a <code>Gemfile</code>
+  </h5>
+  <p>
+    Jekyll treats this particular group of gems(s) with a special love. Any gem
+    included in this group is given a high priority and "loaded" way before
+    Jekyll starts processing the rest of your source directory.
+    <br/>
+    Because of the priority, a gem included here will be activated (even under
```

Comment:
~Because of the priority,~ 

Capitalize the a after removing the priority bit.